internal/provider/const.go

```diff
@@ -30,6 +30,13 @@ const (
 	sectionLabelDescription  = "The label for the section."
 	sectionFieldsDescription = "A list of custom fields in the section."
 
+	filesDescription             = "A list of files attached to the item."
+	fileDescription              = "A file attached to the item."
+	fileIDDescription            = "A UUID for the file."
```

Comment:
`[nitpick]`

To match the same styling of the rest of documentation:
```suggestion
	fileIDDescription            = "The UUID of the file."
```